kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/model/networking/NetworkingPlanner.java

```diff
@@ -118,11 +124,18 @@ private static ClusterIngressNetworkingDefinition clusterIngressNetworkingDefini
                                                                                          List<NodeIdRanges> nodeIdRanges,
                                                                                          @Nullable Tls tls) {
         ClusterIP clusterIP = ingress.getSpec().getClusterIP();
+        LoadBalancer loadBalancer = ingress.getSpec().getLoadBalancer();
         if (clusterIP != null) {
             return new ClusterIPClusterIngressNetworkingDefinition(ingress, cluster, primary, nodeIdRanges, tls);
         }
+        else if (loadBalancer != null) {
+            if (tls == null) {
+                throw new IllegalStateException("Load balancer ingress requires downstream TLS to be provided by the virtualkafkacluster");
```

Comment:
think this is going to blow up the whole proxy reconciliation, I'll check. This should be handled similarly to the conflict exception, reported instead as some reason we couldn't construct the NetworkingModel for that cluster-ingress combo